src/RL/representation/default/defaultstaterepresentation.jl

```diff
@@ -112,7 +111,11 @@ Update the StateRepesentation according to its Type and Featurization.
 """
 function update_representation!(sr::DefaultStateRepresentation, model::CPModel, x::AbstractIntVar)
     update_features!(sr, model)
+    ncon = sr.cplayergraph.numberOfConstraints
+    nvar = sr.cplayergraph.numberOfVariables
+    sr.possibleValuesIdx = map(v -> indexFromCpVertex(sr.cplayergraph, ValueVertex(v)), collect(x.domain))
```

Comment:
Did we specifically test this change ?

Reply:
I don't think we tested it. Maybe we should create an issue for each behavior we want to test, without postponing the merge?